arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -566,153 +572,2046 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved queue description.",
+            "schema": {
+              "$ref": "#/definitions/SubscriptionResource"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ServiceBus/namespaces/{namespaceName}/topics/{topicName}/subscriptions": {
+      "get": {
+        "tags": [
+          "Subscriptions"
+        ],
+        "operationId": "Subscriptions_GetList",
+        "description": "Returns the description for the specified topic",
+        "externalDocs": {
+          "url": "https://msdn.microsoft.com/en-us/library/azure/mt639400.aspx"
+        },
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "namespaceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The namespace name."
+          },
+          {
+            "name": "topicName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The topic name."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved list of Subscriptions.",
+            "schema": {
+              "$ref": "#/definitions/SubscriptionListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SubscriptionCreateOrUpdateParameters": {
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Subscription data center location."
+        },
+        "type": {
+          "type": "string",
+          "description": "Resource Manager type of the resource."
+        },
+        "name": {
+          "type": "string",
+          "description": "Name of the subscription."
+        },
+        "properties": {
+          "$ref": "#/definitions/SubscriptionProperties"
+        }
+      },
+      "required": [
+        "location"
+      ],
+      "description": "Parameters supplied to the CreateOrUpdate Subscription operation."
+    },
+    "SubscriptionProperties": {
+      "properties": {
+        "AccessedAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Last time a there was a receive request to this subscription."
+        },
+        "AutoDeleteOnIdle": {
+          "type": "string",
+          "format": "time-span",
```

Comment:
As I said earlier, Autorest does not support this format. Please provide the format: "duration"